lib/config/fileconf.go

```diff
@@ -854,7 +854,9 @@ func (a *AuthenticationConfig) Parse() (types.AuthPreference, error) {
 }
 
 type UniversalSecondFactor struct {
-	AppID                string   `yaml:"app_id"`
+	AppID string `yaml:"app_id"`
+	// Facets kept only to avoid breakages during Teleport updates.
+	// Webauthn is now used instead of U2F.
 	Facets               []string `yaml:"facets"`
```

Comment:
Should we maybe log a warning and mark this for deletion in Teleport 11 or 12 (by which time nobody should be using U2F anymore)?